eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -10,7 +10,8 @@
     },
     "NodeTestVersion": [ "8.x", "10.x", "12.x", "14.x", "15.x" ],
     "TestType": "node",
-    "TestResultsFiles": "**/test-results.xml"
+    "TestResultsFiles": "**/test-results.xml",
+    "ServiceVersion": ""
```

Comment:
Is there a reason we need to explicitly set an empty service version in each of the base matrix entries?

Reply:
Is it because of:
```
SERVICE_VERSION: $(ServiceVersion)
```
If so we could set a default value for ServiceVersion somewhere higher like https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/variables/globals.yml.

Reply:
Yes, It's because if we didn't the interpolation here: https://github.com/Azure/azure-sdk-for-js/blob/05bde403f3378d3cad86f4efc5daaa3a73573f2c/eng/pipelines/templates/jobs/archetype-sdk-integration.yml#L135 would just literal it, so you end up with `$(ServiceVersion)` as the value for the environment variable. 

I know relatively little about Azure DevOps expressions and variables though so if there's a better approach I'm happy to use it

Reply:
Ah, sorry - replied to your first comment missing the second. That's great - I'll make that change shortly. Thank you!